styleguide/components/11-base/base/_base.scss

```diff
@@ -43,7 +43,7 @@ th > * {
 hr {
   margin: 0 0 $default-line-height;
   clear: both;
-  border: 1px 0 0 solid lighten($default-text-color, 60);
+  border: 1px 0 0 solid ligher('color-dark-gray', 50%);
```

Comment:
Shouldn't we use the tint function here?

Reply:
Yes. changed it.